### PR TITLE
fix(build): fix docker proto files generation

### PR DESCRIPTION
**Pull request by aleem1314**
## Description

Closes: #427 
Ref: [9854](https://github.com/cosmos/cosmos-sdk/pull/9854)

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -334,9 +335,9 @@ devdoc-update:
 
 containerProtoVer=v0.2
 containerProtoImage=tendermintdev/sdk-proto-gen:$(containerProtoVer)
-containerProtoGen=cosmos-sdk-proto-gen-$(containerProtoVer)
-containerProtoFmt=cosmos-sdk-proto-fmt-$(containerProtoVer)
-containerProtoGenSwagger=cosmos-sdk-proto-gen-swagger-$(containerProtoVer)
+containerProtoGen=${PROJECT_NAME}-proto-gen-$(containerProtoVer)
+containerProtoFmt=${PROJECT_NAME}-fmt-$(containerProtoVer)
```
Comment (aleem1314) [inline Makefile:339]:
```suggestion
containerProtoFmt=${PROJECT_NAME}-proto-fmt-$(containerProtoVer)
```

**Review by ryanchristo — APPROVED** [review]
tACK. A couple minor changes are made in `go.mod` and `go.sum` after running `make proto-gen`. We should maybe commit those changes in this pull request.

**Review by amaurym — APPROVED** [review]
nice that's it's a fix that doesn't hardcode "regen-ledger".

TBH I think this fix can be beneficial for other chains, what do you think of copying to the SDK too?

**Comment by robert-zaremba** [comment]
any reason we want to have different container for regen ledger? Now we are using the same image: tendermintdev/sdk-proto-gen:v0.2

**Comment by aleem1314** [comment]
> TBH I think this fix can be beneficial for other chains, what do you think of copying to the SDK too?

 This issue recently addressed on the SDK by this [PR#9854](https://github.com/cosmos/cosmos-sdk/pull/9854).

**Review by atheeshp — APPROVED** [review]

**Pull request merged**